<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет себестоимости продукции</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Отчет</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionOpen"/>
    <addaction name="actionXlsx"/>
    <addaction name="actionDel"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="actionCalc"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Нормы</string>
    </property>
    <addaction name="actionNorm"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="text">
    <string>Загрузить новый из 1С</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionCalc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Расчет</string>
   </property>
  </action>
  <action name="actionXlsx">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Экспорт в xlsx</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удалить отчет</string>
   </property>
  </action>
  <action name="actionNorm">
   <property name="text">
    <string>Нормы расхода</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
